packages/ember-glimmer/lib/helpers/-class.js

```diff
@@ -0,0 +1,25 @@
+import { InternalHelperReference } from '../utils/references';
+import { dasherize } from 'ember-runtime/system/string';
+
+function classHelper({ positional }) {
+  let path = positional.at(0);
+  let className = positional.at(1);
+  let value = path.value();
+
+  if (value === true) {
+    return dasherize(className.value());
```

Comment:
Yeah, I meant that for case `class=fooBar` it should transform into `foo-bar`, but in case `(-class fooBar "fooBar")`, when class passed explicitly by consumer, it should stay `fooBar` (to solve this you could pass dasherized version from the component and do not dasherize in the helper)